cfme/containers/provider/openshift.py

```diff
@@ -192,7 +182,21 @@ def from_config(cls, prov_config, prov_key, appliance=None):
 
         endpoints = {}
         token_creds = cls.process_credential_yaml_key(prov_config['credentials'], cred_type='token')
+
+        with ssh.SSHClient(**{"username": "root",
```

Comment:
**REQUIRED**: I really don't like calling out to SSH during what is essentially the init. I'm not sure what @mfalesni or @jkrocil say, but I think we should be able to instantiate a provider object with calling out to SSH. In fact ding this would mean there would currently be SSH calls during collection. This is usually a big no no.

Reply:
So what do you suggest doing instead, in the previous implementation I overload the ssh_password for each endpoint, and when this certificate was required, it got the certificate via ssh, is this sound better?

I can revert the code to the previous version, but before I do please take a look at it 
 --> https://github.com/ManageIQ/integration_tests/pull/6432/commits/a232af749a7eee66bb54d245909be5a368623c87